exercises/practice/list-ops/.docs/instructions.append.md

```diff
@@ -0,0 +1,8 @@
+# Emacs Lisp Track specific functions
+
+* `list-empty-p` (*given a list, return if the list is empty*)
+* `list-sum` (*given a list of numbers, return the sum of all elements*)
+
+# Hints
```

Comment:
We have a policy of only using one h1 heading

```suggestion
## Hints
```

Reply:
Done